src/server/service/kv.rs

```diff
@@ -611,12 +617,13 @@ impl<T: RaftStoreRouter + 'static, E: Engine, L: LockMgr> Tikv for Service<T, E,
         assert!(!req.get_end_key().is_empty());
 
         let (cb, f) = paired_future_callback();
-        let res = self.storage.async_unsafe_destroy_range(
+        let res = self.gc_worker.async_unsafe_destroy_range(
             req.take_context(),
             Key::from_raw(&req.take_start_key()),
             Key::from_raw(&req.take_end_key()),
             cb,
         );
+        KV_COMMAND_COUNTER_VEC_STATIC.gc.inc();
```

Comment:
I think you'd better put this line (and also that in `future_gc`) in `GCWorker`'s methods, which is just like those storage commands updates this metrics in `Storage`'s methods. By the way, it seems you updated wrong metric here.